golos.vesting/golos.vesting.cpp

```diff
@@ -181,6 +183,14 @@ void vesting::delegate_vesting(account_name sender, account_name recipient, asse
     auto balance_sender = account_sender.find(quantity.symbol.name());
     eosio_assert(balance_sender != account_sender.end(), "Not found token");
 
+    tables::convert_table table_convert(_self, quantity.symbol.name());
+    auto record_convert = table_convert.find(sender);
+    if (record_convert != table_convert.end()) {
+        auto user_balance = balance_sender->available_vesting() -
+                (record_convert->balance_amount - (record_convert->payout_part * record_convert->number_of_payments));
```

Comment:
this looks wrong. `number_of_payments` set to `NUMBER_OF_CONVERSIONS` at object creation, so `user_balance = balance_sender->available_vesting() - 0`, which means it's not limited by `convert` part at this point